includes/service-providers/mailchimp/class-newspack-newsletters-mailchimp-default-footer.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * Service Provider: Mailchimp Default Footer
+ *
+ * @package Newspack
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Usage reports for Mailchimp.
+ */
+class Newspack_Newsletters_Mailchimp_Default_Footer {
+
+	/**
+	 * Initialize hooks.
+	 */
+	public static function init() {
+		add_action( 'admin_footer', [ __CLASS__, 'add_footer_content_to_editor' ] );
+	}
+	/**
+	 * Get the default Mailchimp footer content.
+	 *
+	 * @return string Footer content.
+	 */
+	public static function get_footer_content() {
+
+		$should_append_footer = get_option( 'newspack_mailchimp_auto_append_footer', false );
+		if ( ! $should_append_footer ) {
+			return '';
+		}
+		$manage_preferences_url = 'http://*|UPDATE_PROFILE|*';
+		if ( function_exists( 'wc_get_account_endpoint_url' ) && class_exists( 'Newspack\Reader_Activation' ) && method_exists( 'Newspack\Reader_Activation', 'is_enabled' ) && \Newspack\Reader_Activation::is_enabled() ) {
```

Comment:
NIT: this `class_exists` is redundant, no? Won't the `method_exists` also fail if the class doesn't exist?

```suggestion
		if ( function_exists( 'wc_get_account_endpoint_url' ) && method_exists( 'Newspack\Reader_Activation', 'is_enabled' ) && \Newspack\Reader_Activation::is_enabled() ) {
```